frontend/src/routes/(root)/(logged)/+layout.svelte

```diff
@@ -227,12 +227,16 @@
 	let devOnly = $page.url.pathname.startsWith(base + '/scripts/dev')
 
 	async function loadCopilot(workspace: string) {
-		workspacedOpenai.init(workspace)
+		OpenAi.workspacedOpenai.init(workspace)
+		AnthropicAi.workspacedAnthropic.init(workspace)
+		MistralAi.workspacedMistral.init(workspace)
```

Comment:
can you combine all inits into a single function that you export from copilot/lib?

Reply:
Done!